stylesheets/stylesheet.css

```diff
@@ -778,15 +778,22 @@ form {
 
 .data-box {
   background-color: var(--databox-bg);
-  box-shadow: 0px 5px 10px var(--box-shadow-color);
```

Comment:
This box seems to be used in many other places, and could be the root cause of all these regressions.